<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuggestionWidget</class>
 <widget class="QWidget" name="SuggestionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_suggestion">
       <property name="text">
        <string>Suggestion</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="pushButton_another">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Another</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_suggestion">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_remaining">
     <property name="text">
      <string>Remaining Options</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView_remaining"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
